tests/functional/ContentRepositoryTest.php

```diff
@@ -941,6 +971,39 @@ public function can_sort_contents_list()
         $this->assertEquals(1, $contents[0]['weight']);
         // translations title
         $this->assertEquals('B title', $contents[0]['translations'][0]['title']);
+
+
+        // Ascending
+        $contents = $this->repository->getContentsByLevel(
+            [
+                ['translations.lang', '=', 'en']
+            ],
+            [
+                ['weight', 'ASC'],
+                ['translations.title', 'ASC'],
+            ],
+            null
+        );
+        // weight
+        $this->assertEquals(0, $contents[0]['weight']);
+        // translations title
+        $this->assertEquals('A title', $contents[0]['translations'][0]['title']);
+
+        // Descending
+        $contents = $this->repository->getContentsByLevel(
```

Comment:
When using getContentsByLevel we should tests contents with 'level' field, so that we can check if this function works as expected, and same for getContents - which should ignore contents level while sorting.